registry.json

```diff
@@ -1727,5 +1727,20 @@
       "telegram":"manizzler",
       "operatorName":"manizzler",
       "github":"manizzle"
+   },
+  {
+      "ticker":"ECHO",
+      "poolId":"pool1wrt4jvd6nsxxtjws5lzl54zdjj8793u9jumcltqhsyqcgaa9d52",
```

Comment:
This is not a valid poolid (adapools id).  Your poolid is 70d75931ba9c0c65c9d0a7c5fa544d948fe2c78597378fac17810184